hippo-002.md

```diff
@@ -0,0 +1,244 @@
+# HIPPO-002
+
+```
+HIPPO: 2
+Title: Hemi Economic Model
+Author: Jeff Garzik <jeff@hemi.xyz>
+        Max Sanchez <max@hemi.xyz>
+Comments-Summary: No comments yet.
+Status: Active
+Type: Network
+Created: 2025-10-24
+```
+
+## Background and Motivation
+The Hemi blockchain connects Bitcoin and Ethereum - enabling new  
```

Comment:
Was it intentional to add two spaces to the end of each line?
This prevents GitHub (and most other markdown renderers) from unwrapping the lines, and forces the following line onto a newline.

I think this makes the document harder to read when it is rendered, however it is not a super big issue.

Reply:
Let's get this merged and then I'll do another PR with some formatting tweaks